### DISCUSSION TITLE
Suggestion: Featured artists filter should not be on by default

### DISCUSSION THREAD

**Discussion by LOKSTED** [ppy/osu#22855] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/29303173/224594977-5a87d368-e87f-4b02-b2a7-cb31cc68e27a.png)
Casual players likely won't notice the filter and think that their favorite song doesn't exist on OSU.

Search term of a popular artist yields 0 relevant result with this filter on
![image](https://user-images.githubusercontent.com/29303173/224593058-6552a791-861b-4457-84e8-7317a4ebcb85.png)

The same search result with this filter off reveal a beatmap with 16M plays that was filtered out by default
![image](https://user-images.githubusercontent.com/29303173/224594317-05d651f9-ab55-4ec6-b038-0ab4cc10bd45.png)

**Comment by nagi-desuuu** [comment, marked as answer]
This was intentional; see https://github.com/ppy/osu/pull/21646

**Reply by nagi-desuuu** [reply]
Related: https://github.com/ppy/osu/issues/22009

**Reply by LOKSTED** [reply]
I don't even mind the prompt to turn it off every time.

I just feel like the ability to play with any music regardless of license is a feature unique to OSU that this filter will take away from. Especially with new players

**Reply by peppy** [reply]
See open discussion at https://github.com/ppy/osu/issues/22009, where we still want to improve this (by showing a message similar to the one at song select when a search leads to no results displaying).
